dashboard/app/controllers/maker_controller.rb

```diff
@@ -113,6 +113,16 @@ def schoolchoice
   def login_code
   end
 
+  # GET /maker/display_code
+  # renders a page for users to copy and paste a login key
+  def display_code
+    # Generate encrypted code to display to user
+    auth_user = current_user.authentication_options.find_by_credential_type(AuthenticationOption::GOOGLE)
+    @secret_code = Encryption.encrypt_string_utf8(
+      Time.now.strftime('%Y%m%dT%H%M%S%z') + auth_user['authentication_id'] + auth_user['credential_type']
```

Comment:
we talked about this in-person, but let's double-check (either online or with the rest of the team) whether or not we should include a salt in this encrypted string

Reply:
After our discussion I did some research about how OpenSSL salts and it looks similar to use a password with openssl, and as far as I know, we are using password encryption here. Tomorrow I'm hoping to figure out how to add the salt/password.

Reply:
Per slack discussion - using the encryption key means we don't have to salt. https://codedotorg.slack.com/archives/C0T10HG6N/p1582844178003000